### ISSUE TITLE
[FEATURE REQUEST]: min trading amount

### ISSUE THREAD

**Issue by Milerius** [KomodoPlatform/atomicDEX-Desktop#553]
**Is your feature request related to a problem? Please describe.**

Related to: https://github.com/KomodoPlatform/atomicDEX-API/issues/473#issuecomment-709281914

**Describe the solution you'd like**

Min volume to avoid fees abuse from makers

**Describe alternatives you've considered**

- None

**Additional context**

Depends on #551

**Comment by Milerius** [comment]
Apparently a new field `min_volume` is inside the orderbook entries now to determine min trading_amount

**Comment by cipig** [comment]
here is how it works: https://github.com/KomodoPlatform/atomicDEX-API/issues/473#issuecomment-740379496
on taker side, if you try to start the swap with a volume lower than `min_volume` from orderbook, the order is not matched and turned into a maker order

**Comment by tonymorony** [comment]
@Milerius For visual implementation let's go ahead the same way as @yurii-khi implemented it in mobile GUI:

1) On the maker side "Use custom min volume" setting on the order confirmation popup, similar to the custom protection settings for setprice call min_volume param setup

![image](https://user-images.githubusercontent.com/24797699/106361932-9e587f00-6352-11eb-9fdc-48cff073ad5f.png)

2) On the taker side additional indication in trading widget after order selection:

![image](https://user-images.githubusercontent.com/24797699/106361950-b7f9c680-6352-11eb-8068-fbc12a2fe011.png)

![image](https://user-images.githubusercontent.com/24797699/106361956-c0ea9800-6352-11eb-80b6-45ea35bd720f.png)

**Comment by yurii-khi** [comment]
@tonymorony  Just realized, that on 1st screenshot min volume coin is MORTY, and I'm pretty sure it should be RICK. I've already fixed this in mobile GUI, so maybe it would be better to update screenshot here too.

**Comment by tonymorony** [comment]
great catch @yurii-khi and thanks for a prompt fix! updated first screenshot with a fixed version

**Comment by TheComputerGenie** [comment]
There comes an issue in closing https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/624 as a duplicate of this, or is at least confusing with the wording being used.
In the case that spawned that, if the taker is the one using BTC then why wouldn't they get to be the one deciding what is/isn't acceptable in fees?

**Comment by cipig** [comment]
https://blockchair.com/bitcoin/transaction/03a1270950fccb9ddaa193f2648c387a012b1f57a9e6de21066bd7f2158e4b22
this is maker claiming takerpayment... i guess the 12$ fee payed is deducted from the BTC amount the maker gets
swap is recorded as `BTC        |     0.00886683 | KMD        |    406.54883081`, but maker got only 0.00851110 BTC from the 0.00886683, see tx above
taker also payed fee 16$ in takerpayment https://blockchair.com/bitcoin/transaction/94c04782648fc6d5ccbef7b1aa8bb9e01ac4977f650ae0e03c35b725ac6a7bae
so, 28$ fees were payed in total for a swap of 406 KMD, that is like 15%

**Comment by TheComputerGenie** [comment]
> that is like 15%

And if one decides that is acceptable to them, why would a "decentralized" app deem it not?

**Comment by tonymorony** [comment]
There is still no way to create maker order with `minvolume` on the desktop what is the one of essential features for a profitable LP.

At first, it would be great to cleanly divide selected order match and maker order placing (`setprice`) in place order widget (mobile app can be used for ref)

**Comment by tonymorony** [comment]
Right now it's possible to create a maker order with min vol and also see min volume of maker orders as a taker, so closing this one
